examples/recording_service/pluggable_storage/c/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ add_library(
     FileStorageWriter
         "${CMAKE_BINARY_DIR}/generated/HelloMsg.c"
         "${CMAKE_CURRENT_SOURCE_DIR}/FileStorageWriter.c"
+        "${CMAKE_CURRENT_SOURCE_DIR}/FileStorageUtils.c"
```

Comment:
I'd add also the header files so that they trigger a recompilation if they change.